File: Java/Array-2/centeredAverage.java
public int centeredAverage(int[] nums) {
  int large = nums[0];
  int small = nums[0];
  int sum = 0;
  for(int i : nums) {
    if(large < i) large = i;
    if(small > i) small = i;
    sum += i;
  }
  return (sum - large - small) / (nums.length - 2);
}

File: Python/List-2/centered_average.py
def centered_average(nums):
  min_num = nums[0]
  max_num = nums[0]
  sum = 0
  for i in nums:
    min_num = min(min_num, i)
    max_num = max(max_num, i)
    sum += i
  sum -= (min_num + max_num)
  return (sum / (len(nums) - 2))
